recon_surf/recon-surf.sh

```diff
@@ -540,18 +524,55 @@ echo " " |& tee -a $LF
 echo "================== Creating orig and rawavg from input =========================" |& tee -a $LF
 echo " " |& tee -a $LF
 
+CONFORM_LF=$SUBJECTS_DIR/$subject/scripts/conform.log
```

Comment:
what is the rationale about having this in a separate file? I would find it beneficial to have this information in recon-surf.log right at the start.